<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="adapterBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>871</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="ipmacTable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>861</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SCAN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fakeipText">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logBrowser">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>60</y>
      <width>821</width>
      <height>871</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="iplabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maclabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fakemacText">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="fakeTable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>610</y>
      <width>861</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="arpfakeLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP FAKE INFO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="logLabel">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LOG</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionARP_SCAN"/>
    <addaction name="actionARP_ATTACK"/>
    <addaction name="separator"/>
    <addaction name="actionPhoexina"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionARP_SCAN">
   <property name="text">
    <string>ARP SCAN</string>
   </property>
  </action>
  <action name="actionARP_ATTACK">
   <property name="text">
    <string>ARP ATTACK</string>
   </property>
  </action>
  <action name="actionPhoexina">
   <property name="text">
    <string>Phoexina 0v0</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
